src/containers/UploadButton.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { translate } from '../lib/I18n'
+
+import { uploadFile } from '../actions'
+
+const styles = {
```

Comment:
well, the styles in question are needed for functional reasons, not for aesthetics. Without these styles, the component doesn't work. That's why I think these styles belong to the component.